ecs-cli/modules/compose/cli/ecs/app/service_command.go

```diff
@@ -136,3 +136,42 @@ func deploymentConfigFlags(specifyDefaults bool) []cli.Flag {
 		},
 	}
 }
+
+func loadBalancerFlags(specifyDefaults bool) []cli.Flag {
+	targetGroupArnUsageString := "[Optional] The full Amazon Resource Name (ARN) of the Elastic Load Balancing target group associated with a service."
+	containerNameUsageString := "[Optional] Mandatory if target-group-arn is specified. The container name (as it appears in a container definition)."
+	containerPortUsageString := "[Optional] Mandatory if target-group-arn is specified. The container port to access from the load balancer."
+	loadBalancerNameUsageString := "[Optional] The name of the load balancer."
+	roleUsageString := "[Optional] The name or full Amazon Resource Name (ARN) of the IAM role that allows Amazon ECS to make calls to your load balancer on your behalf. This parameter is required if you are using a load balancer with your service. If you specify the role parameter, you must also specify a target-group-arn, container-name and container-port."
+
+	if specifyDefaults {
+		targetGroupArnUsageString += " Defaults to none"
+		containerNameUsageString += " Defaults to none"
+		containerPortUsageString += " Defaults to none"
+		loadBalancerNameUsageString += " Defaults to none"
+		roleUsageString += " Defaults to ecsServiceRole"
```

Comment:
Does it actually defaults to `ecsServiceRole`?

Also, to specify defaults use `Value` in the cli.StringFlag